EnglishFilter/sections/general_extensions.txt

```diff
@@ -27,7 +27,7 @@ shameless.com#%#//scriptlet("abort-current-inline-script", "$LAB", "/getexoloade
 !
 ! 'PopJSLock'
 ! #%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
-freepornxxxhd.com,givemenflstreams.com,givemenbastreams.com,givememmastreams.com,givemeredditstreams.com,shinden.pl,ozporno.com,area51deportiva.com,xxxfree.watch,linksly.co,animesanka.com,mhometheater.com#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
+tmnlsmvgrkoj.com,xokikdassjjhrn.com,freepornxxxhd.com,givemenflstreams.com,givemenbastreams.com,givememmastreams.com,givemeredditstreams.com,shinden.pl,ozporno.com,area51deportiva.com,xxxfree.watch,linksly.co,animesanka.com,mhometheater.com#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
```

Comment:
So, the general idea about this one, you add the original domain here (not the adserver).
But, you handle it via blocking rules, including Regex. No need for adding domains to the scriptlet.

Reply:
Ok.